server/src/db_connection.rs

```diff
@@ -8,20 +8,29 @@ pub struct DBClient {
     database: Database,
     pub recipe_repo: repository::Recipes,
     pub user_repo: repository::Users,
+    pub meta_repo: repository::MetaRepo,
 }
 
 impl DBClient {
+    /// Create a new mongodb client, containing the repositories used throughout the server.
     pub async fn new(client_uri: impl AsRef<str>, db_name: impl AsRef<str>) -> Result<Self, Error> {
         let client = Client::with_uri_str(client_uri).await?;
         let database = client.database(db_name.as_ref());
         let recipe_repo = repository::Recipes::new(database.collection("recipes"));
         let user_repo = repository::Users::new(database.collection("users"));
+        let meta_repo = repository::MetaRepo::new(database.collection("meta"));
 
         Ok(Self {
             client,
             database,
             recipe_repo,
             user_repo,
+            meta_repo,
         })
     }
+
+    #[doc(hidden)]
+    pub async fn delete_db(&self) {
```

Comment:
Do we use this outside of tests?
If not -> `#[cfg(test)]`?

Reply:
yes, I use it in doctests and those don't observe #[cfg(test)]

Reply:
Those have cfg doctest so you could restrict to either of those

Reply:
fixed in latest commit